src/core/util/latent_see.h

```diff
@@ -120,23 +120,25 @@ class Log {
 
   GPR_ATTRIBUTE_ALWAYS_INLINE_FUNCTION static Log& Get() {
     static Log* log = []() {
-      atexit([] {
-        auto json = log->TryGenerateJson();
-        if (!json.has_value()) {
-          LOG(INFO) << "Failed to generate latent_see.json (contention with "
-                       "another writer)";
-          return;
-        }
-        if (log->stats_flusher_ != nullptr) {
-          log->stats_flusher_(*json);
-          return;
-        }
-        LOG(INFO) << "Writing latent_see.json in " << get_current_dir_name();
-        FILE* f = fopen("latent_see.json", "w");
-        if (f == nullptr) return;
-        fprintf(f, "%s", json->c_str());
-        fclose(f);
-      });
+      if (false) {
```

Comment:
what's this condition used for?

Reply:
oh... I removed the default temporarily and meant to get back to 'what should we do here' -- for now I've added a new API to Log to install the atexit handler for users that want that. I'll probably add it to a bunch of microbenchmarks soon.